ftplugin/crystal.vim

```diff
@@ -32,4 +32,12 @@ setlocal commentstring=#\ %s
 let &cpo = s:save_cpo
```

Comment:
Please move `setlocal errorformat= ...` config before this line.  Please see `:help use-cpo-save`.  This is a guard for legacy Vim which doesn't support line continuation.